ssbls12.py

```diff
@@ -32,6 +35,10 @@ def __add__(self, other):
         assert type(other) is SS_BLS12_381
         return SS_BLS12_381(add(self.m1, other.m1), add(self.m2, other.m2))
 
+    def __sub__(self, other):
```

Comment:
I don't understand the maths behind the pairing curves. Intuitively, this felt correct.